ntp-proto/src/algorithm/kalman/peer.rs

```diff
@@ -90,6 +90,22 @@ struct AveragingBuffer {
     next_idx: usize,
 }
 
+/// Approximation of 1 - the chi-squared cdf with 1 degree of freedom
+/// source: https://en.wikipedia.org/wiki/Error_function
+fn chi_1(chi: f64) -> f64 {
+    const P: f64 = 0.3275911;
+    const A1: f64 = 0.254829592;
+    const A2: f64 = -0.284496736;
+    const A3: f64 = 1.421413741;
+    const A4: f64 = -1.453152027;
+    const A5: f64 = 1.061405429;
+
+    let x = (chi / 2.).sqrt();
+    let t = 1. / (1. + P * x);
+    (A1 * t + A2 * t * t + A3 * t * t * t + A4 * t * t * t * t + A5 * t * t * t * t * t)
```

Comment:
are there disadvantages to using https://doc.rust-lang.org/std/primitive.f64.html#method.powi here?

Reply:
I didn't find it much cleaner (because of brackets) but other than that no.